Common/Data/Auxiliary/FactorFileRow.cs

```diff
@@ -306,10 +306,9 @@ public string ToCsv(string source = null)
         {
             source = source == null ? "" : $",{source}";
             return $"{Date.ToStringInvariant(DateFormat.EightCharacter)}," +
-                   Invariant($"{Math.Round(PriceFactor, 6).Normalize()},") +
-                   Invariant($"{Math.Round(SplitFactor, 7).Normalize()},") +
-                   Invariant($"{Math.Round(ReferencePrice, 2).Normalize()}") +
-                   $"{source}";
+                   $"{PriceFactor.Normalize()}," +
```

Comment:
I think we wanted to keep the `Invariant` here -- this ensures that when we generate the string it uses the invariant culture instead of the machine's current culture setting.